rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf

```diff
@@ -55,7 +55,7 @@ SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:941012,phase:2,pass,nolog,skipAf
 # support this feature at runtime.
 #
 SecRule REQUEST_FILENAME "!@validateByteRange 20, 45-47, 48-57, 65-90, 95, 97-122" \
-    "id:941010,\
+    "id:94.0.1-dev,\
```

Comment:
```suggestion
    "id: 941010,\
```

Reply:
Will the added whitespace upset the linter?

Reply:
Thanks, changed.